vMenuServer/MainServer.cs

```diff
@@ -222,6 +222,18 @@ public MainServer()
                     Debug.WriteLine($"\n\n^1[vMenu] [ERROR] ^7Your addons.json file contains a problem! Error details: {ex.Message}\n\n");
                 }
 
+                // check extras file for errors
+                string extras = LoadResourceFile(GetCurrentResourceName(), "config/extras.json") ?? "{}";
```

Comment:
Could you add an example `extras.json` file in the config folder for a non-existing vehicle? That way we can safely provide a default example for people to modify and use.

Reply:
Added sample file